examples/tof-viewer/include/ADIToFRecorder.h

```diff
@@ -165,13 +165,15 @@ class ADIToFRecorder {
     uint16_t *frameDataLocationDEPTH = nullptr;
     uint16_t *frameDataLocationXYZ = nullptr;
     uint16_t *frameDataLocationIR = nullptr;
+    uint16_t *frameDataLocationHeader = nullptr;
     aditof::FrameDetails m_frameDetails;
     bool m_saveBinaryFormat = false;
     bool m_finishRecording = true;
     bool _stopPlayback = false;
     int currentPBPos = 0;
     int m_numberOfFrames;
     int fileSize = 0;
+    uint16_t sizeOfHeader;
```

Comment:
Please add prefix "m_".
I know there are other variables that are not using it, but at least the newly added code to comply with the unwritten coding style.
It would make things a bit easier to read when variables are encountered through the .cpp file.

Reply:
Fixed.